src/app/@ansyn/ansyn/modules/overlays/overlay-status/overlay-status.component.ts

```diff
@@ -168,6 +169,14 @@ export class OverlayStatusComponent implements OnInit, OnDestroy, IEntryComponen
 			})
 		);
 
+	@AutoSubscription
+	onClickOutSide = () => this.isAfterViewInit.pipe(switchMap(() => this.clickOutsideService.onClickOutside({monitor: this.closeElementAnnotation.nativeElement}).pipe(
```

Comment:
Very good.

I don't think we need two pipe()?
This is not considered good practice.
Why not with one pipe(), like:

pipe(
  switchMap(...),
  filter(..)
  ...
}?